src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -526,6 +526,7 @@ public enum SyntaxKind : ushort
         Utf8StringLiteralToken = 8520,
         Utf8SingleLineRawStringLiteralToken = 8521,
         Utf8MultiLineRawStringLiteralToken = 8522,
+        BadRazorContentToken = 8523,
```

Comment:
do we want some sort of razor section?

Reply:
I don't think so? I think the single token should be fine.

Reply:
wfm.  what i was imaginging though was that there might be a few types of razor kinds we detected.  like razor comments, bad razor tokens.  perhaps a couple different razor tokens.  

--

Ah, i now see why i don't like this name.  i want somethnig to indicate it's a `@:`.  So perhaps BadRazorAtColonToken.  Or perhaps BadRazorLineToken.  Something that really conveys this is the token we produce when we see `@:` and go to the end of the line.

I feel moderately strongly :)

Reply:
See, I want exactly the opposite: I _don't_ want to break out the various things we do for error recovery for razor. What if next year razor adds a new thing that we want to recover for, are we going to need to break that out separately? Or what if Razor removes the need for us to handle this? I'd rather it just be "this is something C# doesn't expect", with no further specificity.

Reply:
_Or_ what if C# decides that `@:` is a cool syntax to use for something next year, so we add an `AtColonToken`, and now we have this vestigal `BadRazorAtSignColonToken`.

Reply:
Razor is already in the business of inspecting characters, so that doesn't feel oogie to me :slightly_smiling_face:. I'd prefer to keep the C# api flexible here.